### DISCUSSION TITLE
A way to hook the modal popup event

### DISCUSSION THREAD

**Discussion by xHeinrich** [livewire/livewire#1101] [4. General]
Would be nice to have a javascript event we can hook when the error modal pops up. Example it will popup if you idle a page too long and the csrf token expires, would be nice in this case if I can do a full page reload. Would be nice if the event can pass the http status code with it so we can react differently to different errors.

**Comment by mblarsen** [comment]
Possibly related #537 and #1031

**Comment by xHeinrich** [comment]
Fixed in 1.3.0 with -  Better handling of session expiries. By default, a confirm() message asks the user if they would like to refresh the page. You are able to override this behaviour using the new livewire.onError() method. #1146
